jmix-flowui/flowui-themes/src/main/resources/META-INF/resources/themes/jmix-lumo/component/jmix-horizontal-menu.css

```diff
@@ -14,41 +14,54 @@
  * limitations under the License.
  */
 
-.jmix-navigation-menu-bar {
+.jmix-horizontal-menu {
```

Comment:
<img width="184" alt="Screenshot 2023-12-26 at 15 36 39" src="https://github.com/jmix-framework/jmix/assets/6077149/0b23b000-a178-4250-8506-c31b2a4667e4">

even if an icon is set, an extra space is present

Reply:
Also, re-style `<a>` so that it takes the whole space, otherwise it's hard to click